cli/cli-utils/src/packageIsInstallable.ts

```diff
@@ -38,7 +38,9 @@ export function packageIsInstallable (
       if (pmVersion && pnpmVersion && pmVersion !== pnpmVersion) {
         const msg = `This project is configured to use v${pmVersion} of pnpm. Your current pnpm is v${pnpmVersion}`
         if (opts.packageManagerStrict) {
-          throw new PnpmError('BAD_PM_VERSION', msg)
+          throw new PnpmError('BAD_PM_VERSION', msg, {
+            hint: 'If you want to make this check less strict, set the package-manager-strict setting to false or the COREPACK_ENABLE_STRICT env variable to 0',
```

Comment:
Actually, I think this language is not direct enough.

```suggestion
            hint: 'If you want ignore this error, set the package-manager-strict setting to false or the COREPACK_ENABLE_STRICT env variable to 0',
```